### DISCUSSION TITLE
How can it be possible to edit`linuxParameters` for a container definition?

### DISCUSSION THREAD

**Discussion by mariocesar** [aws/copilot-cli#2626] [Q&A]
I see where the template for CF defines de containerDefinition -> https://github.com/aws/copilot-cli/blob/77983421537d9f34575655ab8cba309f30610f33/templates/workloads/services/backend/cf.yml#L44-L54 

I can't see any definition to override linuxParameters.

There is a clean way or prefered way to customize containers definitions?

I would like to set `tmpfs` or `initProcessEnabled`

**Comment by iamhopaul123** [comment]
Hello @mariocesar. We don't plan to expose every field of task definition in our manifest. Instead, we would add a section to our current manifest to allow [task definition override](https://github.com/aws/copilot-cli/issues/2588). Hopefully it would help for this case!

**Reply by mariocesar** [reply]
Thanks, what you propose solves the issue. Is that already available ?
